callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -196,8 +200,9 @@ public void onError(CallStatsErrors error, String errMsg) {
 	 * Send call stats bridge status update.
 	 *
 	 * @param bridgeStatusInfo the bridge status info
+	 * @throws UnsupportedEncodingException 
 	 */
-	public void sendCallStatsBridgeStatusUpdate(BridgeStatusInfo bridgeStatusInfo) {
+	public void sendCallStatsBridgeStatusUpdate(BridgeStatusInfo bridgeStatusInfo) throws UnsupportedEncodingException {
```

Comment:
This is risky, wouldn't this make so that applications that depend on the library would fail to compile if they are not catching UnsupportedEncodingException? (ex. this would be non-backwards compatible)